adsmp/app.py

```diff
@@ -160,3 +217,59 @@ def get_msg_status(self, msg):
                 return 'active'
         else:
             return 'unknown'
+        
+    
+
+    def reindex(self, solr_docs, batch_insert, batch_update, solr_urls):
+        """Sends documents to solr and to Metrics DB.
```

Comment:
Similarly, when the Solr schema changes I would like to avoid having to write all the records to Metrics.